relayer/package.json

```diff
@@ -17,7 +17,7 @@
   "dependencies": {
     "@coral-xyz/anchor": "^0.28.0",
     "@coral-xyz/borsh": "^0.28.0",
-    "@lightprotocol/zk.js": "file:../light-zk.js",
+    "@lightprotocol/zk.js": "0.3.2-alpha.13",
```

Comment:
We need this to import by path for testing. What about doing relayer release branches which have the actual version number and will be deployed?